src/org/jetbrains/plugins/scala/compiler/HydraCredentialsManager.scala

```diff
@@ -0,0 +1,83 @@
+package org.jetbrains.plugins.scala.compiler
+
+import java.io.{File, FileNotFoundException}
+import java.nio.charset.StandardCharsets
+import java.util.{Base64, Properties}
+import javax.crypto.spec.SecretKeySpec
+
+import com.intellij.ide.passwordSafe.PasswordSafe
+import com.intellij.openapi.application.PathManager
+import com.intellij.openapi.diagnostic.Logger
+import com.intellij.util.proxy.PropertiesEncryptionSupport
+
+import scala.util.{Failure, Try}
+
+/**
+  * @author Maris Alexandru
+  */
+object HydraCredentialsManager {
```

Comment:
I guess this whole class has to migrate to the `PasswordSafe` API, as discussed on Slack? They sent [this link](https://github.com/JetBrains/intellij-community/blob/master/platform/credential-store/readme.md).